src/requests/consumption.js

```diff
@@ -1629,7 +1629,7 @@ const publishProgram = function async(programData) {
 					}
 					resourceToUpdate.push({
 						_id: fetchDetails?.result?.published_id,
-						updateBody,
+						updateBody: updateBody.data,
```

Comment:
@adithyadinesh0412 change updateBody to updatePayload
 const updatePayload = await updateSolutionTemplate(fetchResult?.data);
 
 updateBody to updateData